app/src/main/java/com/babylon/wallet/android/presentation/status/assets/lsu/LSUDialogContent.kt

```diff
@@ -273,17 +273,18 @@ private fun LSUResourceValue(
                 maxLines = 1
             )
 
-            val xrdPriceFormatted = remember(price, amount) {
+            val xrdPrice = remember(price, amount) {
                 if (amount != null) {
-                    price?.xrdPriceFormatted(amount)
+                    price?.xrdPrice(amount)
                 } else {
                     null
                 }
             }
-            if (xrdPriceFormatted != null) {
-                FiatBalance(
-                    fiatPriceFormatted = xrdPriceFormatted,
-                    style = RadixTheme.typography.body2HighImportance
+            if (xrdPrice != null) {
+                FiatBalanceView(
+                    fiatPrice = xrdPrice,
+                    textStyle = RadixTheme.typography.body2HighImportance,
+                    isLoading = false // TODO
```

Comment:
I would remove the `isLoading` param from all `FiatBalanceView` as it is not needed.